src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -249,6 +249,7 @@ class NotebooksContent extends Component {
       acceptStyle: { cursor: 'copy' },
       rejectStyle: { cursor: 'no-drop' },
       ref: this.uploader,
+      onDropRejected: () => reportError('Error uploading notebook', 'The selected file is not a ipynb notebook file. To import a notebook, upload a file with a .ipynb extension.'),
```

Comment:
Maybe make the short text 'Not a valid notebook'?

Reply:
Do you mean to make the error `reportError('Not a valid notebook', 'The selected file is not a ipynb notebook file. To import a notebook, upload a file with a .ipynb extension.')` ?

Error heading:
> **Not a valid notebook**

click on details shows:
> The selected file is not a ipynb notebook file. To import a notebook, upload a file with a .ipynb extension.